pkg/cluster/arooperator.go

```diff
@@ -10,22 +10,13 @@ import (
 )
 
 func (m *manager) ensureAROOperator(ctx context.Context) error {
-	dep, err := deploy.New(m.log, m.env, m.doc.OpenShiftCluster, m.arocli, m.extensionscli, m.kubernetescli)
-	if err != nil {
-		m.log.Errorf("cannot ensureAROOperator.New: %s", err.Error())
-		return err
-	}
-	err = dep.CreateOrUpdate(ctx)
+	err := deploy.New(m.log, m.env, m.doc.OpenShiftCluster, m.arocli, m.extensionscli, m.kubernetescli, m.dynamichelper).CreateOrUpdate(ctx)
```

Comment:
Please let's not use chaining in Go code. Especially in such a long line. It is hard to follow: took me a minute to understand why we call `deploy.New()`, but the error is about `CreateOrUpdate`.

Reply:
done :)